nqeit-iceberg-flink-extension/src/main/java/org/projectnessie/integtests/flink/Flink.java

```diff
@@ -29,4 +29,14 @@
 @Target(ElementType.PARAMETER)
 @Retention(RetentionPolicy.RUNTIME)
 @Inherited
-public @interface Flink {}
+public @interface Flink {
+
+  /**
+   * the nessie client configuration is derived from all system properties that start with
+   * "nessie.client."
+   *
+   * <p>config values can be overriden selectively by providing another prefix here (i.e.
+   * "nessie.readonly.client")
+   */
+  String nessieClientOverrideSystemPropertyPrefix() default "";
```

Comment:
Suggestion for brevity: `prefix() default "nessie.client." ` plus `overrides() default ""`. The info about system property loading and applying them to Nessie Clients can probably be in the javadoc. WDYT?

Reply:
i briefly thought about it before but it would repeat the default value in multiple places, which also felt odd.

as to the verbosity of the name: i felt like the annotation could in the future hold more configuration options, not all of which are related to the nessie client.